conf/local.conf.sample

```diff
@@ -94,40 +98,3 @@ PATCHRESOLVE = "noop"
 RM_OLD_IMAGE = "1"
 INHERIT += "rm_work"
 CONF_VERSION = "1"
-
-export SSH_AUTH_SOCK
```

Comment:
some users may still want this line?

Reply:
I think I agree with @costanic in that keeping this in would probably be fine.